### ISSUE TITLE
Bug on the kH values

### ISSUE THREAD

**Issue by lipelopesoliveira** [lipelopesoliveira/flames-ads#18]
### 🐛 Describe the bug

There is an error on the kH and Qst calculation by the wisdom method.

### Versions

Python 3.11.13
absl-py @ file:///home/conda/feedstock_root/build_artifacts/absl-py_1751547525079/work
annotated-types==0.7.0
array-api-compat==1.12.0
ase==3.25.0
asttokens @ file:///home/conda/feedstock_root/build_artifacts/asttokens_1733250440834/work
astunparse @ file:///home/conda/feedstock_root/build_artifacts/astunparse_1736248061654/work
bibtexparser @ file:///home/conda/feedstock_root/build_artifacts/bibtexparser_1734879298321/work
bracex==2.6
Brotli @ file:///D:/bld/brotli-split_1749229870877/work
certifi @ file:///home/conda/feedstock_root/build_artifacts/certifi_1754231422783/work/certifi
cffi @ file:///D:/bld/cffi_1725560706917/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1746214863626/work
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1733218098505/work
comm @ file:///home/conda/feedstock_root/build_artifacts/comm_1733502965406/work
contourpy==1.3.2
cycler @ file:///home/conda/feedstock_root/build_artifacts/cycler_1733332471406/work
dargs==0.4.10
debugpy @ file:///D:/bld/bld/rattler-build_debugpy_1752827138/work
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1740384970518/work
deepmd-kit==3.1.0
Deprecated==1.2.18
exceptiongroup @ file:///home/conda/feedstock_root/build_artifacts/exceptiongroup_1746947292760/work
executing @ file:///home/conda/feedstock_root/build_artifacts/executing_1745502089858/work
filelock==3.13.1
flatbuffers @ file:///home/conda/feedstock_root/build_artifacts/python-flatbuffers_1739279199749/work
fonttools @ file:///D:/bld/fonttools_1752722652722/work
fsspec==2024.6.1
future @ file:///home/conda/feedstock_root/build_artifacts/future_1738926421307/work
gast @ file:///home/conda/feedstock_root/build_artifacts/gast_1733299680490/work
google-pasta @ file:///home/conda/feedstock_root/build_artifacts/google-pasta_1733852499742/work
greenlet==3.2.3
grpcio @ file:///C:/b/abs_98_4ysdoqy/croot/grpc-split_1742490255707/work
h2 @ file:///home/conda/feedstock_root/build_artifacts/h2_1738578511449/work
h5py @ file:///C:/b/abs_29off2a8q5/croot/h5py_1750863017528/work
hpack @ file:///home/conda/feedstock_root/build_artifacts/hpack_1737618293087/work
hyperframe @ file:///home/conda/feedstock_root/build_artifacts/hyperframe_1737618333194/work
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1733211830134/work
importlib_metadata @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_importlib-metadata_1747934053/work
importlib_resources @ file:///home/conda/feedstock_root/build_artifacts/importlib_resources_1736252299705/work
ipykernel @ file:///D:/bld/ipykernel_1719845595208/work
ipython @ file:///D:/bld/bld/rattler-build_ipython_1751470649/work
ipython_pygments_lexers @ file:///home/conda/feedstock_root/build_artifacts/ipython_pygments_lexers_1737123620466/work
jedi @ file:///home/conda/feedstock_root/build_artifacts/jedi_1733300866624/work
Jinja2==3.1.4
joblib @ file:///home/conda/feedstock_root/build_artifacts/joblib_1748019130050/work
jupyter_client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1733440914442/work
jupyter_core @ file:///D:/bld/jupyter_core_1748333031907/work
keras @ file:///home/conda/feedstock_root/build_artifacts/keras_1747722781675/work
kiwisolver @ file:///D:/bld/kiwisolver_1751493881237/work
latexcodec @ file:///home/conda/feedstock_root/build_artifacts/latexcodec_1592937263153/work
Markdown @ file:///home/conda/feedstock_root/build_artifacts/markdown_1750360292101/work
markdown-it-py @ file:///home/conda/feedstock_root/build_artifacts/markdown-it-py_1733250460757/work
MarkupSafe @ file:///D:/bld/markupsafe_1733219614059/work
matplotlib==3.10.1
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1733416936468/work
mdurl @ file:///home/conda/feedstock_root/build_artifacts/mdurl_1733255585584/work
mendeleev==1.1.0
ml-dtypes @ file:///D:/bld/ml_dtypes_1736538804541/work
monty @ file:///home/conda/feedstock_root/build_artifacts/monty_1741071865066/work
mpi4py @ file:///D:/bld/bld/rattler-build_mpi4py_1750855745/work
mpmath @ file:///home/conda/feedstock_root/build_artifacts/mpmath_1733302684489/work
munkres==1.1.4
namex @ file:///home/conda/feedstock_root/build_artifacts/namex_1748346368839/work
narwhals @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_narwhals_1753814733/work
nest_asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1733325553580/work
networkx==3.3
numpy @ file:///D:/bld/numpy_1732314323952/work/dist/numpy-2.0.2-cp311-cp311-win_amd64.whl#sha256=256c08f2badfbb3090cf87d2a238847df5e794797acf8346288c8ee02f558811
opt_einsum @ file:///home/conda/feedstock_root/build_artifacts/opt_einsum_1733687912731/work
optree @ file:///D:/bld/optree_1748442476125/work
packaging @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_packaging_1745345660/work
palettable @ file:///home/conda/feedstock_root/build_artifacts/palettable_1733997590492/work
pandas @ file:///D:/bld/pandas_1752081717412/work
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1733271261340/work
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1733327343728/work
pillow==11.0.0
platformdirs @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_platformdirs_1746710438/work
plotly @ file:///home/conda/feedstock_root/build_artifacts/plotly_1751025118896/work
prompt_toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1744724089886/work
protobuf @ file:///D:/bld/protobuf_1741124581122/work/bazel-bin/python/dist/protobuf-5.29.3-cp311-abi3-win_amd64.whl#sha256=fc6f951fa9dad094b03d76e7ff3e8040a212ea3d461c179f17c7096177be30f2
psutil @ file:///D:/bld/psutil_1740663127374/work
pure_eval @ file:///home/conda/feedstock_root/build_artifacts/pure_eval_1733569405015/work
pybtex @ file:///home/conda/feedstock_root/build_artifacts/pybtex_1751014981901/work
pycparser @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_pycparser_1733195786/work
pydantic==2.11.7
pydantic_core==2.33.2
pyfiglet==0.8.post1
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1750615794071/work
pymatgen @ file:///D:/bld/pymatgen_1745527059750/work
pyparsing @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_pyparsing_1753873557/work
PySocks @ file:///D:/bld/pysocks_1733217287171/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_python-dateutil_1751104122/work
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1742920838005/work
pywin32 @ file:///D:/bld/bld/rattler-build_pywin32_1752564079/work
PyYAML @ file:///D:/bld/pyyaml_1737454660454/work
pyzmq @ file:///D:/bld/pyzmq_1749898448924/work
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1749498106507/work
rich @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_rich_1743371105/work/dist
ruamel.yaml @ file:///D:/bld/ruamel.yaml_1749479944820/work
ruamel.yaml.clib @ file:///D:/bld/ruamel.yaml.clib_1728724614818/work
scipy @ file:///C:/bld/scipy-split_1751160069174/work/dist/scipy-1.16.0-cp311-cp311-win_amd64.whl#sha256=501ec7a40e8c93ed7a3b8aff6384c5eeb90071e24d500550ebdade18abebeeee
six @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_six_1753199211/work
spglib @ file:///D:/bld/spglib_1741599972733/work
SQLAlchemy==2.0.41
stack_data @ file:///home/conda/feedstock_root/build_artifacts/stack_data_1733569443808/work
sympy==1.13.3
tabulate @ file:///home/conda/feedstock_root/build_artifacts/tabulate_1733589744265/work
tensorboard @ file:///home/conda/feedstock_root/build_artifacts/tensorboard_1733820672324/work/tensorboard-2.18.0-py3-none-any.whl#sha256=107ca4821745f73e2aefa02c50ff70a9b694f39f790b11e6f682f7d326745eab
tensorboard_data_server @ file:///D:/bld/tensorboard-data-server_1728639813727/work/tensorboard_data_server-0.7.0-py3-none-any.whl#sha256=a0ee9efb4274a8eb541b5d39494b020a6654522a4d20e16631e5ea685eac5862
tensorflow_cpu==2.18.1
termcolor @ file:///home/conda/feedstock_root/build_artifacts/termcolor_1746039537955/work
torch==2.7.1+cu128
torchaudio==2.7.1+cu128
torchvision==0.22.1+cu128
tornado @ file:///D:/bld/tornado_1748003321906/work
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1735661334605/work
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1733367359838/work
typeguard==4.4.4
typing-inspection==0.4.1
typing_extensions @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_typing_extensions_1751643513/work
tzdata @ file:///home/conda/feedstock_root/build_artifacts/python-tzdata_1742745135198/work
uncertainties @ file:///home/conda/feedstock_root/build_artifacts/uncertainties_1745274325800/work
unicodedata2 @ file:///D:/bld/unicodedata2_1736692523011/work
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1750271362675/work
wcmatch==10.1
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1733231326287/work
Werkzeug @ file:///home/conda/feedstock_root/build_artifacts/werkzeug_1733160440960/work
win_inet_pton @ file:///D:/bld/win_inet_pton_1733130564612/work
wrapt @ file:///D:/bld/wrapt_1736869488076/work
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1749421620841/work
zstandard==0.23.0
